tests/env/test_env.py

```diff
@@ -12,17 +12,20 @@
 from conda.base.context import context, reset_context
 from conda.common.serialize import yaml_round_trip_load
 from conda.core.prefix_data import PrefixData
-from conda.exceptions import CondaHTTPError, EnvironmentFileNotFound
-from conda.models.match_spec import MatchSpec
-from conda.testing import CondaCLIFixture, PathFactoryFixture
-from conda.testing.integration import package_is_installed
-from conda_env.env import (
+from conda.env.env import (
     VALID_KEYS,
     Environment,
     from_environment,
     from_file,
-    load_from_directory,
 )
+from conda.exceptions import CondaHTTPError, EnvironmentFileNotFound
+from conda.models.match_spec import MatchSpec
+from conda.testing import CondaCLIFixture, PathFactoryFixture
+from conda.testing.integration import package_is_installed
+
+# Note: The conda_env.env module is getting deprecated in 24.3,
+#       this is the only place that it's utilized in the codebase
```

Comment:
```suggestion
# Note: The conda_env.env module is getting deprecated in 24.3,
#       this is the only place that the load_from_directory function
#       utilized in the codebase.
```
Is that correct?

Reply:
Yes, [that's correct](: https://github.com/search?q=repo%3Aconda%2Fconda%20load_from_directory&type=code).

Reply:
Thank you!